ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/methods/engine/EngineNewPayloadV3.java

```diff
@@ -54,24 +44,12 @@ public String getName() {
   }
 
   @Override
-  protected ValidationResult<RpcErrorType> validateParamsAndForkSupported(
-      final Object reqId,
-      final EnginePayloadParameter payloadParameter,
-      final Optional<List<String>> maybeVersionedHashParam,
-      final Optional<Bytes32> maybeParentBeaconBlockRoot) {
-
+  protected ValidationResult<RpcErrorType> validateParameter(
+      final EnginePayloadParameter payloadParameter) {
     if (payloadParameter.getBlobGasUsed() == null || payloadParameter.getExcessBlobGas() == null) {
       return ValidationResult.invalid(RpcErrorType.INVALID_PARAMS, "Missing blob gas fields");
-    } else if (maybeVersionedHashParam == null) {
-      return ValidationResult.invalid(
-          RpcErrorType.INVALID_PARAMS, "Missing versioned hashes field");
-    } else if (maybeParentBeaconBlockRoot.isEmpty()) {
-      return ValidationResult.invalid(
-          RpcErrorType.INVALID_PARAMS, "Missing parent beacon block root field");
-    }
-    if (payloadParameter.getTimestamp() < cancunTimestamp) {
-      return ValidationResult.invalid(RpcErrorType.UNSUPPORTED_FORK, "Fork not supported");
+    } else {
+      return ValidationResult.valid();
```

Comment:
we have lost some of the checks here. We need to check that the versioned hash params and the parent beacon block root are available when we are calling the v3 RPC.

Reply:
They are checked in the parent abstract class, via the validateParams method.

Reply:
syncResponse is calling the much more robust validateBlobs, but it doesn't look like validateBlobs enforces that the parameters are present.  Does the spec for v3 require `expectedBlobVersionedHashes` to be present even if the list is empty?